spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -56,6 +60,21 @@ public void postProcessBeanFactory(
 		this.beanFactory = beanFactory;
 	}
 
+	private Storage getStorage() {
```

Comment:
A getter that changes the state of the object?

Reply:
This is `private` method, so harmless